tests/generic_config_updater/test_lo_interface.py

```diff
@@ -73,6 +74,10 @@ def setup_env(duthosts, rand_one_dut_hostname, lo_intf):
         check_show_ip_intf(
             duthost, DEFAULT_LOOPBACK,
             [lo_intf["ipv6"].lower()], ["Vrf"], is_ipv4=False)
+
+        # Loopback interface removal will impact default route. Reload to recover routes.
+        config_reload(duthost)
```

Comment:
Config reload may cost 3-5 minutes. Is there a quicker way to recover?

Reply:
will try restart bgp first then reload if fail